src/psyclone/psyir/nodes/acc_directives.py

```diff
@@ -376,31 +376,6 @@ def fields(self):
                     fld_list.append(arg)
         return fld_list
 
-    @property
-    def scalars(self):
```

Comment:
Just to double check. Do the LFric OpenACC implmentation that I think is being developed and is currently not in the unit-tests will need this functionality?

Reply:
I don't know TBH. I figured if it isn't being used now for GOcean then it's probably not needed. We can always put it back if we need it in future.